src/components/Card/Card.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import {activeOpacity, metrics} from '../../helpers/metrics'
+import type {ITheme} from '../../theme'
+import type {ViewProps} from 'react-native'
+
+export type CardProps = {
+  onPress?: () => void
+  children?: React.ReactNode
+  bgColor?: string
+  style?: ViewProps
+  testId?: string
+}
+
+const Card: React.FC<CardProps> = ({onPress, children, bgColor, style, testId}) => (
+  <View
```

Comment:
i think you should name it  CardContainer or CardWrapper ... would be better instead of view which actually is a TouchableOpacity

Reply:
ok I updated it.